pkg/qemu/qemu.go

```diff
@@ -540,7 +540,8 @@ func Cmdline(ctx context.Context, cfg Config) (exe string, args []string, err er
 			}
 		}
 	}
-	if !strings.Contains(string(features.CPUHelp), strings.Split(cpu, ",")[0]) {
+	// `qemu-system-ppc64 -help` does not show "max", but it is actually accepted
+	if cpu != "max" && !strings.Contains(string(features.CPUHelp), strings.Split(cpu, ",")[0]) {
```

Comment:
Technically it should probably be:

```suggestion
	if (*y.arch != limayaml.PPC64LE || cpu != "max") && !strings.Contains(string(features.CPUHelp), strings.Split(cpu, ",")[0]) {
```

But I guess we just assume that `max` is valid for all CPUs now.